packages/validator/src/spec/PwaValidatorSpec.ts

```diff
@@ -99,5 +114,193 @@ describe('PwaValidator', () => {
           .toBe('https://developers.google.com/speed/pagespeed/insights/' +
               '?url=https%3A%2F%2Fexample.com%2F');
     });
+
+    // FCP Tests
+    it('FCP is PASS when 0', async () => {
+      const psiResult = mockPsiResult(0.8, 1.0, {
+        firstContentfulPaint: 0,
+        largestContentfulPaint: 0,
+        maxPotentialFID: 0,
+        cumulativeLayoutShift: 0,
```

Comment:
Could you use the `{ ... oldArray }` syntax here? I'm not 100%, but I think this bit could be rewritten as:

```
const psiResult = mockPsiResult(0.8, 1.0, {
  firstContentfulPaint: 0,
  ...WEB_VITALS_SCORES,
});
```
So it makes it a bit more obvious which lines we care about and which ones are just the default.

(Maybe `firstContentfulPaint` needs to come after the `WEB_VITALS_SCORES`, I'm not 100% sure. Do tell me if you try it.)

Reply:
It does work (though `WEB_VITALS_SCORES` needs to come first). Thanks for the suggestion!

Reply:
done